src/core/sensor/qgsiodevicesensor.h

```diff
@@ -261,6 +261,17 @@ class CORE_EXPORT QgsSerialPortSensor : public QgsIODeviceSensor
      */
     void setPortName( const QString &portName );
 
+    /**
+    * Returns the baudrate of the serial port the sensor connects to.
+    */
+    QString baudrate() const;
```

Comment:
It's preferable to align our function names with that of Qt. In this case QSerialPort sets the function name to baudRate/setBaudRate, let's do that too.
```suggestion
    QSerialPort::BaudRate baudRate() const;
```